### DISCUSSION TITLE
Excel problem with CoolProp Mixture

### DISCUSSION THREAD

**Discussion by cliffbradford** [CoolProp/CoolProp#2452] [Q&A]
New user in Excel and I'm trying to calculate properties of an air water vapor mixture using PropsSI. I've already run some cases with HAPropsSI but I may need properties at temps above the validity of HAPropsSI. WHenI try to run a case with an air & water mixture I get an error.
I've entered this formula in a cell
=PropsSI("CPMASS","T",CONVERT($A4,"C","K"),"P",CONVERT($B4,"atm","Pa"),"HEOS::"&MixtureString($S$2:$S$3,AD12:AD13))

where: 
A4 = 15
B4 = 1
S2:s3 = "Air":"Water"
AD12:AD13 = .985:.185

I get the following error:

Initialize failed for backend: "HEOS", fluid: "Air&Water" fractions "[ 0.[9855880381](tel:(985)%20588-0381), 0.0144119619 ]"; error: Could not match the binary pair [7732-18-5,AIR.PPF] - for now this is an error. : PropsSI("CPMASS","T",288.15,"P",101325,"HEOS::Air[.985588038076329]&Water[1.44119619236714E-02]")

**Comment by henningjp** [comment]
Couple of things here.

Binary interaction parameters must exist for the pure fluids in the mixture.  CoolProp defines these for many fluids, but not the two fluids specified because air is a pseudo-pure fluid.

You **_might_** be better off specifying the actual components of air (N2,O2,Ar,CO2) plus Water, however this becomes really hard because of the disparity in boiling points.  Air is a cryogenic fluid and Water is not.  If the state point is well into the vapor region for water this might work, but I've not had success doing this.
